src/components/common/NameRequestInfo.vue

```diff
@@ -1,8 +1,9 @@
 <template>
   <div id="name-request-info">
+    <!-- display name request info -->
     <template v-if="getNameRequestNumber">
       <!-- Name Request -->
-      <div class="section-container mb-n8">
```

Comment:
Instead of shrinking this component (to absorb whitespace to the next component), I updated the next component to not add top whitespace.